pkg/export/alloy/traces.go

```diff
@@ -40,11 +55,15 @@ func (tr *tracesReceiver) provideLoop() (pipe.FinalFunc[[]request.Span], error)
 	}
 	return func(in <-chan []request.Span) {
 		// Get user attributes
-		traceAttrs, err := otel.GetUserSelectedAttributes(tr.attributes)
+		traceAttrs, err := tr.getConstantAttributes()
 		if err != nil {
 			slog.Error("error fetching user defined attributes", "error", err)
 		}
 
+		if tr.spanMetricsEnabled {
+			traceAttrs[attr.SkipSpanMetrics] = struct{}{}
+		}
+
```

Comment:
Looks duplicate (already inside `getConstantAttributes`)

Reply:
Ah nice catch, I moved it there so I can make the test and forgot to take out out.